src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,278 @@
+    //given
+    Menu request = menu();
+    MenuGroup menuGroup = menuGroup();
+    Product product = product();
+
+    when(menuGroupRepository.findById(any())).thenReturn(Optional.of(menuGroup));
+    //when
+    List<Product> productList = Arrays.asList(product, product);
+    when(productRepository.findAllByIdIn(any())).thenReturn(productList);
+
+    //then
+    assertThatThrownBy(() -> {
+      menuService.create(request);
+    }).isInstanceOf(IllegalArgumentException.class);
+  }
+
+  @Test
+  @Order(5)
+  @DisplayName("메뉴에 올라간 상품의 수량이 음수면 IllegalArgumentException 예외 발생")
+  void checkQuantity() {
+    //given
+    Menu request = menu();
+    MenuGroup menuGroup = menuGroup();
+    MenuProduct menuProduct = menuProduct();
+    Product product = product();
+    List<Product> productList = Arrays.asList(product, product);
+
+    when(menuGroupRepository.findById(any())).thenReturn(Optional.of(menuGroup));
+    when(productRepository.findAllByIdIn(any())).thenReturn(productList);
+
+    //when
+    menuProduct.setQuantity(NEGATIVE_NUM);
+
+    //then
+    assertThatThrownBy(() -> menuService.create(request))
+            .isInstanceOf(IllegalArgumentException.class);
+  }
+
+  @Test
+  @Order(6)
+  @DisplayName("메뉴 가격은 메뉴의 상품들 가격의 합보다 비싸면, IllegalArgumentException 예외 발생")
+  void comparePrice() {
+    //given
+    Menu request = menu();
+    Product product = product();
+    List<Product> productList = Collections.singletonList(product);
+
+    when(menuGroupRepository.findById(any())).thenReturn(Optional.of(request.getMenuGroup()));
+    when(productRepository.findAllByIdIn(any())).thenReturn(productList);
+
+    //when
+    product.setPrice(BigDecimal.ZERO);
+    when(productRepository.findById(any())).thenReturn(Optional.of(product));
+
+    //then
+    assertThatThrownBy(() -> menuService.create(request))
+            .isInstanceOf(IllegalArgumentException.class);
+  }
+
+  @Test
+  @Order(7)
+  @DisplayName("가게 점주는 가격을 변경할 수 있습니다.")
+  void changePriceInMenu() {
+    //given
+    BigDecimal changePrice = BigDecimal.valueOf(1000);
+    Menu request = menu();
+    request.setPrice(changePrice);
+    when(menuRepository.findById(any())).thenReturn(Optional.of(menu()));
+
+    //then
+    Menu changePriceMenu = menuService.changePrice(UUID, request);
+    assertThat(changePriceMenu.getPrice()).isEqualTo(changePrice);
+  }
+
+  @Order(8)
+  @ParameterizedTest
+  @MethodSource("menuPriceNullAndMinus")
+  @DisplayName("변경하려는 메뉴의 가격은 0원 이상입니다.")
+  void changePriceIsPositiveNum(BigDecimal price) {
+    //given
+    Menu request = menu();
+
+    //when
+    request.setPrice(price);
+
+    //then
+    assertThatThrownBy(() -> menuService.changePrice(UUID, request))
+            .isInstanceOf(IllegalArgumentException.class);
+  }
+
+  @Test
+  @Order(9)
+  @DisplayName("메뉴 가격은 메뉴의 상품들 가격의 합보다 비싸면 IllegalArgumentException 예외 발생")
+  void compareChangePrice() {
+    //given
+    Menu request = menu();
+    Menu menu = menu();
```

Comment:
테스트 코드만 봐서 테스트의 의도를 알수 있으면 좋을것 같아요.

```suggestion
    Menu request = 메뉴_가격_2000원();
    Menu menu = 메뉴_가격_1500원();
```

Reply:
바로 의도를 알 수 있는 방법이 있군요!! 좋은 정보 감사합니다. ☺️